app/views/static_pages/api.html.haml

```diff
@@ -72,29 +72,33 @@
                   target: "_blank") + "."
 
     %p
+      - party1 = Party.all.sample.name
 
       For example if a tactical voting site wishes to recommend a user
-      to vote for the Liberal Democrat candidate in the Somerton and Frome
+      to vote for (randomly chosen)
+      = party1
+      candidate in the Aberdeen North
       constituency, they could use this outbound link:
 
-      - url1 = swap_url(willing_party_name: "liberal_democrats",
-                        constituency_name: "Somerton and Frome")
-
-      %pre
-        %code
-          = link_to url1, url1
+    %p
+      - url1 = swap_url(willing_party_name: party1.parameterize.gsub('-', '_'),
+                        constituency_name: "Aberdeen North")
+      = link_to url1, url1
 
-      This one would also work:
+    %p
+      This one would also work, but rather than specify the tactical vote, instead specifies the user’s preferred party (randomly chosen). This time their constituency is York Outer (specified by ONS code).
```

Comment:
For consistency and completeness, why not embed `party2` here like you did with `party1` above?

Reply:
Done